src/main/webapp/i18n/de/tutorialGroupImport.json

```
@@ -6,19 +6,19 @@
             "importButton": "Übunsgruppen importieren",
             "dialogTitle": "Übungsgruppen importieren",
             "explanations": {
-                "description": "Dieser Dialog kann verwendet werden, um Übungsgruppen und deren Registrierungen aus einer CSV-Datei zu importieren. Dabei werden Übungsgruppen erstellt, wenn sie noch nicht existieren. Die Studierende werden von ihrer aktuellen Übungsgruppe abgemeldet und in der neuen Übungsgruppe registriert.",
-                "requirements": "Jede Zeile muss immer einen Wert für den Übungsgruppen Titel enthalten. Wenn nicht nur Übungsgruppen erstellt werden sollen, sondern auch die Registrierungen von Studierenden, muss die Zeile einen Wert für die Immatrikulationsnummer oder für den Benutzernamen enthalten. Dies ist erforderlich, um den Studierenden zu identifizieren. Werte für den Nachnamen und Vornamen sind optional. Die erste Zeile muss die Spaltennamen angeben!"
+                "description": "Dieses Dialogfeld ermöglicht es, Tutoriumsgruppen und Studentenregistrierungen aus einer CSV-Datei zu importieren. Wenn eine Tutoriumsgruppe noch nicht existiert, wird sie erstellt. Wenn Studenten in der CSV-Datei aufgeführt sind, werden sie von ihrer aktuellen Tutoriumsgruppe in die neue, in der CSV-Datei angegebene Gruppe verschoben.",
+                "requirements": "Jede Zeile muss den Titel der Tutoriumsgruppe enthalten. Um Studenten zu registrieren, muss jede Zeile entweder eine Matrikelnummer oder einen Login-Wert enthalten. Vor- und Nachname sind optional. Die oberste Zeile der CSV-Datei muss die Überschriften angeben."
```

```suggestion
                "requirements": "Jede Zeile muss den Titel der Tutoriumsgruppe enthalten. Um Studierende zu registrieren, muss jede Zeile entweder eine Matrikelnummer oder einen Login-Namen enthalten. Vor- und Nachname sind optional. Die oberste Zeile der CSV-Datei muss die Spaltennamen angeben."
```
- Guidelines 
- ‘Login-Wert’ sounds pretty unusual, ‘Login-Name’ would be more intuitive to me
- I would keep ‘Spaltennamen’ is translation for field names. Reasoning: It is one CSV header row having field names.

---

done, but used anmeldename instead of login name